### PR TITLE
Model and Repo for the ops list

### PR DESCRIPTION
**Pull request by Prashantjagwani**
Hey @moretanjames Could you review this PR too..🙂
The endpoint is not ready for the ops list but i have created the model and repo as per the dummy data. Below is the dummy json response for that.

`[
    {
        "id": 1,
        "publicId": "01-01",
        "opTitle": "Eat lunch",
        "opDate": "2021-08-09T14:23:40.161Z",
        "multidayEndDate": null,
        "acceptedOperators": 8,
        "operatorsNeeded": 2,
        "opQuantity": 6.0,
        "opTermsUnit": "Hour",
        "opPay": 200.0,
        "timeZoneId": "Some ID",
        "estTotalHours": 8.0,
        "breakHours": 2.0,
        "opSkillImage": "https://s3.us-east-2.amazonaws.com/veryable-public-assets/skills-v2/large/forklift_default_large.png",
        "opSkillDisplayName": "Fork-lifting",
        "estMinPerUnit": null,
        "business": {
            "businessName": "Some cool business",
            "businessHorizontalLogo": "https://veryable-files-production.s3.us-east-2.amazonaws.com/business_logos/small/revive.png",
            "parentBusinessHorizontalLogo": "https://veryable-files-production.s3.us-east-2.amazonaws.com/business_logos/small/benefithub.png"
        }
    },
    {
        "id": 2,
        "publicId": "01-02",
        "opTitle": "Eat lunch",
        "opDate": "2021-08-10T14:23:40.161Z",
        "multidayEndDate": "2021-08-11T14:23:40.161Z",
        "acceptedOperators": 8,
        "operatorsNeeded": 0,
        "opQuantity": 6.0,
        "opTermsUnit": "Hour",
        "opPay": 200.0,
        "timeZoneId": "Some ID",
        "estTotalHours": 8.0,
        "breakHours": 2.0,
        "opSkillImage": "https://s3.us-east-2.amazonaws.com/veryable-public-assets/skills-v2/large/forklift_default_large.png",
        "opSkillDisplayName": "Fork-lifting",
        "estMinPerUnit": null,
        "business": {
            "businessName": "Some cool business",
            "businessHorizontalLogo": "https://veryable-files-production.s3.us-east-2.amazonaws.com/business_logos/small/revive.png",
            "parentBusinessHorizontalLogo": "https://veryable-files-production.s3.us-east-2.amazonaws.com/business_logos/small/benefithub.png"
        }
    },
    {
        "id": 3,
        "publicId": "01-03",
        "opTitle": "Eat lunch",
        "opDate": "2021-08-11T14:23:40.161Z",
        "multidayEndDate": "2021-08-12T14:23:40.161Z",
        "acceptedOperators": 8,
        "operatorsNeeded": 2,
        "opQuantity": 6.0,
        "opTermsUnit": "Hour",
        "opPay": 200.0,
        "timeZoneId": "Some ID",
        "estTotalHours": 8.0,
        "breakHours": 2.0,
        "opSkillImage": "https://s3.us-east-2.amazonaws.com/veryable-public-assets/skills-v2/large/forklift_default_large.png",
        "opSkillDisplayName": "Fork-lifting",
        "estMinPerUnit": null,
        "business": {
            "businessName": "Some cool business",
            "businessHorizontalLogo": "https://veryable-files-production.s3.us-east-2.amazonaws.com/business_logos/small/revive.png",
            "parentBusinessHorizontalLogo": "https://veryable-files-production.s3.us-east-2.amazonaws.com/business_logos/small/benefithub.png"
        }
    }
]`

### CODE REVIEW

File: app/src/main/java/com/veryableops/commandcenter/models/opslist/Ops.kt [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    val operatorsNeeded: Int?,
+    val opQuantity: Double?,
+    val opTermsUnit: String?,
+    val opPay: Double?,
+    val timeZoneId: String?,
+    val estTotalHours: Double?,
+    val breakHours: Double?,
+    val opSkillImage: String?,
+    val opSkillDisplayName: String?,
+    val estMinPerUnit: Any?,
```
Comment (moretanjames) [inline app/src/main/java/com/veryableops/commandcenter/models/opslist/Ops.kt:22]: This is Of type `Any` but Im pretty sure its actually a double or an Int. Also you have all of these as nullable but not all are. I know the id isn't as well as publicId, opTitle, opDate, operatorsNeeded, opQuantity, timeZoneId, opSkillImage, opSkillDisplayName, opBusiness and opPay arent. I know what I said to Subham yesterday but this object I think is going to be much more stable than the one he was working on so I don't think we have to be as drastic with the nullable.

File: app/src/main/java/com/veryableops/commandcenter/models/opslist/OpsBusiness.kt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package com.veryableops.commandcenter.models.opslist
+
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class OpsBusiness(
+    val businessName: String?,
```
Comment (moretanjames) [inline app/src/main/java/com/veryableops/commandcenter/models/opslist/OpsBusiness.kt:7]: Same comment as before

File: app/src/main/java/com/veryableops/commandcenter/repos/opslist/OpsListApiService.kt [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import retrofit2.Response
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface OpsListApiService {
+
+    @GET("")
+    suspend fun getOps(
+        @Query("district") district: String?,
+        @Query("id") id: String?
```
Comment (moretanjames) [inline app/src/main/java/com/veryableops/commandcenter/repos/opslist/OpsListApiService.kt:13]: I dont think well need this Id

**Review by moretanjames — CHANGES_REQUESTED** [review]
Looking good just a few comments

File: app/src/main/java/com/veryableops/commandcenter/models/opslist/Ops.kt (hunk 1 above)
Comment (Prashantjagwani) [reply app/src/main/java/com/veryableops/commandcenter/models/opslist/Ops.kt]: Changed estMinPerUnit type to Double. Made few fields non-nullable.

File: app/src/main/java/com/veryableops/commandcenter/repos/opslist/OpsListApiService.kt (hunk 3 above)
Comment (Prashantjagwani) [reply app/src/main/java/com/veryableops/commandcenter/repos/opslist/OpsListApiService.kt]: Removed id as params.

File: app/src/main/java/com/veryableops/commandcenter/models/opslist/OpsBusiness.kt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+package com.veryableops.commandcenter.models.opslist
+
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class OpsBusiness(
+    val businessName: String,
+    val businessHorizontalLogo: String,
```
Comment (moretanjames) [inline app/src/main/java/com/veryableops/commandcenter/models/opslist/OpsBusiness.kt:8]: the two logos are nullable

**Review by moretanjames — CHANGES_REQUESTED** [review]
Just the one thing then itll be merged

File: app/src/main/java/com/veryableops/commandcenter/models/opslist/OpsBusiness.kt (hunk 4 above)
Comment (Prashantjagwani) [reply app/src/main/java/com/veryableops/commandcenter/models/opslist/OpsBusiness.kt]: Done.👍🏻

**Pull request merged**